newIDE/app/src/GameDashboard/Feedbacks/GameFeedback.js

```diff
@@ -222,119 +245,216 @@ const GameFeedback = ({ i18n, authenticatedUser, game }: Props) => {
     [loadFeedbacksAndBuilds]
   );
 
+  const openOptionsContextMenu = (event: MouseEvent) => {
+    if (contextMenu.current) {
+      contextMenu.current.open(event.clientX, event.clientY);
+    }
+  };
+
+  const markAllCommentsAsProcessed = async (i18n: I18nType) => {
+    if (!profile || isMarkingAllAsProcessed) return;
+    try {
+      setIsMarkingAllAsProcessed(true);
+      await Promise.all(
+        displayedFeedbacksArray
+          .filter(comment => !comment.processedAt)
+          .map(comment =>
+            updateComment(getAuthorizationHeader, profile.id, {
+              gameId: comment.gameId,
+              commentId: comment.id,
+              processed: true,
+            })
+          )
+      );
```

Comment:
Does it send a request for each message?

Reply:
yes it does, I think it was the most pragmatic.
We could modify the backend services to only accept a gameId and loop through them if we realise this improves performance.
But as it's not really a critical part of the app, let's go with this and improve that later